src/data/people.json

```diff
@@ -1,23 +1,23 @@
 [
     {
-        "title": "Taylor Otwell",
-        "known_for": "Creator of Laravel",
+        "title": "Jeffrey Way",
```

Comment:
Please add new entry at the bottom of the list, not at the top of the list.

Reply:
sure.